tests/src/test/kotlin/com/android/declarative/infra/DeclarativeTest.kt

```diff
@@ -45,6 +45,7 @@ pluginManagement {
 
 plugins {
     id 'com.android.experiments.declarative.settings' version '0.0.1'
+    id 'com.android.experiments.declarative.project' version '0.0.1' apply false
```

Comment:
Does this mean we'll need to add settings and project plugins to settgins.gradle file after this change?

Reply:
no, I removed it, like I said in the previous comment, it is now added by the settings plugin in the root project.